app/src/main/java/protect/card_locker/importexport/Exporter.kt

```diff
@@ -0,0 +1,25 @@
+package protect.card_locker.importexport
+
+import android.content.Context
+import android.database.sqlite.SQLiteDatabase
+import java.io.IOException
+import java.io.OutputStream
+
+/**
+ * Interface for a class which can export the contents of the database
+ * in a given format.
+ */
+interface Exporter {
+    /**
+     * Export the database to the output stream in a given format.
+     *
+     * @throws IOException
```

Comment:
Not your fault, but this comment seems outdated.


```suggestion
     * @throws IOException, InterruptedException
```